app/app/urls.py

```diff
@@ -454,6 +455,7 @@
         name='issue_details_new2'
     ),
     re_path(r'^funding/details/?', dashboard.views.bounty_details, name='funding_details'),
+    re_path(r'^issue/(?P<bounty_id>\d+)', dashboard.views.bounty_details, name='issue_details_new4'),
```

Comment:
couldn't we remove the old ones  issue_details_new2 / is that still needed ?

Reply:
I would not remove this now, see my comment above.